wenwenkong/wenwenkong.github.io
# Fix build error

# Changes
- Resolve pending merge conflicts
- Change from `figure.html` to `figure.liquid`
- Remove a bunch of example posts
- Don't show profiles page

# Testing
Follow instructions from https://github.com/wenwenkong/wenwenkong.github.io/blob/master/INSTALL.md
1. Run `docker compose pull`
2. Run `docker compose up`, see build succeed
3. Visit http://127.0.0.1:8080 and verify the webpages
